wandb/sdk/launch/sweeps/utils.py

```diff
@@ -151,19 +153,23 @@ def construct_scheduler_entrypoint(
             )
             return None
 
-    entrypoint = ["wandb", "scheduler", "WANDB_SWEEP_ID"]
     args = [
         "--queue",
         f"{queue!r}",
         "--project",
         project,
         "--num_workers",
         f"{num_workers}",
+        "--sweep_type",
+        f"{sweep_type}",
     ]
 
     if author:
         args += ["--author", author]
 
+    if num_previous_runs:
```

Comment:
Why not just have the scheduler look up the number of previous runs when it's started up? Feels weird that this is an arg